scripts/docker_build.sh

```diff
@@ -20,5 +21,10 @@
 GIT_ROOT=$(git rev-parse --show-toplevel)
 GIT_REVISION=$(git describe --abbrev=10 --always --tags --dirty)
 
-echo "Setting ORT_VERSION to $GIT_REVISION."
-DOCKER_BUILDKIT=1 docker build -f $GIT_ROOT/Dockerfile -t ort --build-arg ORT_VERSION=$GIT_REVISION $GIT_ROOT
+echo "Setting ORT_VERSION to $GIT_VERSION."
+docker buildx build -f "$GIT_ROOT"/Dockerfile \
+    -t "$ORT_DOCKER" \
```

Comment:
There is no `ORT_DOCKER` variable; so this should probably stay at "ort".

Reply:
If someone pass ORT_DOCKER=ort_my_custom the container will be created  as ort_my_custom, otherwise will be used as ort

Reply:
So let's say you want a custom image and want specia version of python and want to keep the pristine docker image ?
`PYTHON_VERSION=3.9.9 ORT_DOCKER=ort_python399 ./scripts/docker_build.sh`